lib/resources/filesystem.rb

```diff
@@ -2,30 +2,104 @@ module Inspec::Resources
   class FileSystemResource < Inspec.resource(1)
     name 'filesystem'
     supports platform: 'linux'
+    supports platform: 'windows'
     desc 'Use the filesystem InSpec resource to test file system'
     example "
+      'filesystem' currently not supported for Linux
```

Comment:
Is this accurate? We say we support it above.

Reply:
hi @miah 
when you ask for i think its a bit missleading. The resource 'filesystem' works on Windows and Linux. But the resource itself has an Option .filesystem this is currently only available on Windows.

```
      describe filesystem('/') do 
        its('size') { should be >= 32000 } 
        its('filesystem') { should eq false }   <-- this is Always 'false' on linux 
      end 

      describe filesystem('c:') do 
        its('size') { should be >= 90 } 
        its('filesystem') { should eq 'NTFS' } <-- for Windows it return the "filesystem" for the resource
     end
```

I am fine if you want to update the documentation according to a better wording,

Reply:
Since we're adding this attribute could we name it something else? I'd go with `type` to make it match our other FS related resource [mount](https://www.inspec.io/docs/reference/resources/mount/).

```
describe filesystem('/') do
  its('type') { should eq 'NTFS' }
end
```

Reply:
update to 'type' . Thanks for the good feedback

Reply:
Thanks! I think we can remove that 'filesystem is not supported' part of the description too.